shared.c

```diff
@@ -498,6 +498,30 @@ void DeploySober(string p="", string v=""){
 	trUnitTeleport(trVectorQuestVarGetX(v),trVectorQuestVarGetY(v),trVectorQuestVarGetZ(v));
 }
 
+/*
+Shows the Chat Log
+*/
+void ChatLogShow(int p = 1){
+	if (trCurrentPlayer() == p) {
+		trChatHistoryClear();
+	}
+	int x = peekModularCounterNext("chat" + p + "Log");
+	for(i=x;<11){
+		trChatSendToPlayer(0, p, trStringQuestVarGet("chat" + p + "Log" + i));
+	}
+	for(i=1;<x){
+		trChatSendToPlayer(0, p, trStringQuestVarGet("chat" + p + "Log" + i));
+	}
```

Comment:
for (i =  12; >0) {
        trChatSendToPlayer(0, p, trStringQuestVarGet("chat" + p + "Log" + modularCounterNext("chat"+p+"log")));
}